src/javaharness/java/arcs/nativeparticles/EchoParticle.java

```diff
@@ -1,6 +1,7 @@
 package arcs.nativeparticles;
 
 import arcs.api.NativeParticle;
+import arcs.api.NativeParticleImpl;
 
-public class EchoParticle implements NativeParticle {
+public class EchoParticle extends NativeParticleImpl implements NativeParticle {
```

Comment:
I think we don't need the `implements` part any more b/c `NativeParticleImpl` implements that?

Reply:
removed.